public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -325,6 +450,13 @@ function AddAnomalyDetector({ embeddable, closeFlyout, mode, setMode }) {
                   ))}
                 </EuiFormFieldset>
                 <EuiSpacer size="m" />
+                {mode === 'existing' && (
+                  <AssociateExisting
+                    embeddableVisId={embeddable.vis.id}
+                    selectedDetector={selectedDetector}
+                    setSelectedDetector={setSelectedDetector}
+                  ></AssociateExisting>
+                )}
                 {mode === 'create' && (
```

Comment:
Similar to the conditional on line 830, we can do the same here for these 2 exclusive scenarios.

Reply:
mode can also be 'associated'

Reply:
What is the difference between `existing` and `associated`? I'm thinking if we can have these in constants with details about each scenario it would be helpful.

Reply:
existing is the one with the button to associate an existing detector flyout and associated is the flyout with the list of all currently associated detectors with ability to unlink. I can add some comments and constants, will do that in my final commit after fixing the dependency issue